tools/test

```diff
@@ -39,21 +66,31 @@ EOF
 coverage=
 files=branch
 fix=
+platform=sloppy
 suites=()
```

Comment:
tiny nit: clearest if these are kept in the same order as the cases that operate on them, below